src/core/packet.h

```diff
@@ -161,7 +170,7 @@ typedef struct QUIC_LONG_HEADER_V1 {
     //uint8_t PacketNumber[PnLength];
     //uint8_t Payload[0];
 
-} QUIC_LONG_HEADER_V1;
+} QUIC_LONG_HEADER_V1, QUIC_LONG_HEADER_V2;
```

Comment:
Since the V1 and V2 structs work for both, should we just call them `QUIC_LONG_HEADER_V1V2` (and any other similar types)?

Reply:
Probably. Thoughts?

Reply:
It'll be consistent, but cluttered-looking, IMO. What if V3 doesn't change anything except the long header again? then everything will be `_V1V2V3`. I'm not sure what solution is best here.

Reply:
after offline discussion, I'll revert to `_V1` and include a comment at the definition which explains that the struct is used for versions 1 and 2.